package.json

```diff
@@ -128,21 +130,30 @@
     "url": "https://github.com/bahmutov/cypress-react-unit-test.git"
   },
   "dependencies": {
+    "@babel/core": "7.4.5",
+    "@babel/preset-env": "7.4.5",
     "@babel/plugin-transform-modules-commonjs": "7.10.4",
     "@cypress/code-coverage": "3.8.1",
     "@cypress/webpack-preprocessor": "5.4.5",
-    "@mdx-js/loader": "^1.6.16",
```

Comment:
agree with next and framer-motion and i18next - these are all for our examples, thus should be dev dependencies